src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -84,13 +84,14 @@ codeLens
     -> CodeLensParams
     -> IO (Either ResponseError (List CodeLens))
 codeLens _lsp ideState CodeLensParams{_textDocument=TextDocumentIdentifier uri} = do
+    pid <- getPid
```

Comment:
It seems fragile that we are adding the prefix in multiple places independently (e.g. here and in `main`). How about, we add a field to `IdeOptions` with the prefix and then fill that in in `main` and then have something like `prefixCommand :: IdeOptions -> Text -> Text` that we use here and in `main`?

Reply:
I did not put it in IdeOptions, as we do not have those whole creating the prefixed command in Main.hs.  But the pid is stable, so wrapped it in a function used in the two places.